src/lib/__tests__/api-test.js

```diff
@@ -182,4 +195,41 @@ describe('API helper', () => {
             });
         });
     });
+
+    describe('addResponseInterceptor', () => {
+        it('should add an http response interceptor', () => {
+            const responseInterceptor = sinon.stub();
+            api.addResponseInterceptor(responseInterceptor);
+
+            expect(api.client.interceptors.response.handlers[0].fulfilled).to.equal(responseInterceptor);
+            api.ejectInterceptors();
```

Comment:
We should probably be creating a new `api` instance for each test rather than modifying a single shared instance.